examples/alias_startup.py

```diff
@@ -6,10 +6,10 @@
 """
 import argparse
```

Comment:
I'm pretty sure that both ``argparse`` and ``pyparsing`` are unused imports within this example and should be removed